content/docs/command-reference/init.md

```diff
@@ -130,15 +127,19 @@ include:
 - There is no need to keep the history at all, e.g. having a deployment
   automation like running a data pipeline using `cron`.
 
-In this mode DVC features that depend on Git being present are not available -
-e.g. managing `.gitignore` files on `dvc add` or `dvc run` to avoid committing
-DVC-tracked files into Git, or `dvc diff` and `dvc metrics diff` that accept
-Git-revisions to compare, etc.
+In this mode, DVC features related to versioning are not available. For example
+automatic creation and updating of `.gitignore` files on `dvc add` or `dvc run`,
+as well as `dvc diff` and `dvc metrics diff`, which require Git revisions to
+compare.
 
 DVC sets the `core.no_scm` config option value to `true` in the DVC
-[config](/doc/command-reference/config) when it is initialized this way. It
-means that even if the project was Git-tracked already or Git is initialized in
-it later, DVC keeps operating in the detached from Git mode.
+[config](/doc/command-reference/config) when initialized this way. This means
+that even if the project is tracked by Git, or if Git is initialized in it
+later, DVC will keep operating detached from Git in this project.
+
+> Note that like with nested <abbr>repositories</abbr> and `--subdir` projects,
```

Comment:
so, if behavior is the same, do we actually need this note?

Reply:
- [ ] Maybe not. This isolation and nested situation is too complicated... I should probably make a separate sub-section to cover it after explaining all the 3 kinds of projects (typical or git-enabled, --subdir, --no-scm).

Can we extract to another issue/PR and get this merged maybe? I have some other changes I also want to throw in on another regular updates PR 😬

Reply:
Let's remove it then, create a ticket and discuss the changes in advance?

Reply:
Removed. I have all the text already in another branch though so I'll just open a draft PR to discuss, if you don't mind... ⌛